core/variant/variant_utility.cpp

```diff
@@ -484,6 +484,14 @@ struct VariantUtilityFunctions {
 		return str;
 	}
 
+	static inline String error_string(int error) {
```

Comment:
I wonder if we can use `Error` instead of `int` as the parameter type for `error`. We should also check if this appears in the generated documentation.

Reply:
Seems to work. But using Error as argument type in `@GlobalScope.xml` still only displays `int` in the documentation

Reply:
Static Checks error out, because Error is an undefined type in `@GlobaScope.xml`? Locally this worked fine. Reverting that for now